include/juicyc/preprocessor_impl.cc

```diff
@@ -0,0 +1,72 @@
+//Created by FENG, H
+#include "preprocessor_impl.h"
+#include <sstream>
+#include <string>
+#include <fstream>
+#include <map>
+#include <regex>
+using namespace std;
+namespace juicy {
+
+	void Preprocessor_impl::process_define()
```

Comment:
Please take a look at google code style (I left a compact version in resource before). We should use CamelCase for class and function name in this project.